style.css

```diff
@@ -259,17 +262,35 @@ section{
       margin:2rem 2.5rem;
       display: block;
   }
-
+    /* cart page */
+    .cart-checkout{
+      flex-direction: column;
+   }
+   .cart-checkout .apply-coupan {
+    width: 100%;
+  }
+  .cart-checkout .cart-total {
+    width: 100%;
+  }
 }
 
 @media (max-width:450px){
```

Comment:
Ideal max-width for mobile screen is 576 as per my knowledge. You can do your own research.

Reply:
ok, I dont know , I will do this change